homeassistant/components/services.yaml

```diff
@@ -0,0 +1,15 @@
+persistent_notification:
```

Comment:
Following @MartinHjelmare 's comment, instead of creating a dedicated folder, an option could be including the domain and service name:

```yaml
persistent_notification: #domain
  create: # service name
    description: 
    fields:
```